woocommerce/woocommerce
# [CYS on Core] Add the sample product badge before the product name to mark placeholder products.

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #45690.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Make sure you have the WooCommerce Beta Tester plugin installed and activated
2. Under Tools > WCA Test Helper > Features, make sure customize-store is enabled
3. Under Tools > WCA Test Helper > Tools, click on Reset Customize Your Store and **Delete all products**
4. Head over to WooCommerce > Home > Customize your store
5. Click on "Start designing" and proceed to the Pattern Assembler
6. Click on Done
7. On the transitional page, click on "Go to products"
8. You should land on Products > All Products screen
9. Make sure you can visualize the "sample" badge on the right side of the product title:

<img width="1868" alt="Screenshot 2024-03-18 at 21 24 46" src="https://github.com/woocommerce/woocommerce/assets/15730971/b1653bfb-f662-4d51-b9cb-62cc8e80fbcb">

10. Create a new product: ensure the "Sample" badge is not displayed for your newly created product.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Add the sample product badge before the product name to mark placeholder products.

#### Comment 

</details>
